doc/spec/estimation/dml.rst

```diff
@@ -553,26 +586,25 @@ Multiple Outcome, Multiple Treatments
 In settings like demand estimation, we might want to fit the demand of multiple products as a function of the price of each one of them, i.e. fit the matrix of cross price elasticities. The latter can be done, by simply setting :math:`Y` to be the vector of demands and :math:`T` to be the vector of prices. Then we can recover the 
 matrix of cross price elasticities as:
 
-.. code-block:: python3
-    :caption: Cross-Price Elasticities
+.. testcode::
 
+    from sklearn.linear_model import MultiTaskElasticNet
     est = LinearDMLCateEstimator(model_y=MultiTaskElasticNet(alpha=0.1),
                                  model_t=MultiTaskElasticNet(alpha=0.1))
     est.fit(Y, T, None, W)
 
     # a_hat[i,j] contains the elasticity of the demand of product i on the price of product j
     a_hat = est.const_marginal_effect()
 
+.. todo:: The following example requires lightning before we can run it as testcode
```

Comment:
@vasilismsr 

I've enabled all code in this file to run as doctests, except for this one example, because it requires `lightning`.  Do you think it's worth adding lightning as a test-time dependency just for this one example?

Reply:
For some weird reason I really like this example :) I believe lightning is also pip installable

Reply:
Okay, added lightning when building docs and re-enabled this example (which had an issue: shape of X passed to const_marginal_effect was not correct)